lib/spack/spack/relocate.py

```diff
@@ -57,9 +57,11 @@ def get_existing_elf_rpaths(path_name):
     """
     if platform.system() == 'Linux':
         command = Executable(get_patchelf())
+        rpaths = None
         output = command('--print-rpath', '%s' %
                          path_name, output=str, err=str)
-        return output.rstrip('\n').split(':')
```

Comment:
I plan on looking at this more tomorrow, but FYI the changes in this method would not change the behavior (i.e. it will return the same thing it has returned before).